pkg/controller/cronjob/utils_test.go

```diff
@@ -84,9 +85,13 @@ func TestGetJobFromTemplate2(t *testing.T) {
 	if len(job.ObjectMeta.Labels) != 1 {
 		t.Errorf("Wrong number of labels")
 	}
-	if len(job.ObjectMeta.Annotations) != 1 {
+	if len(job.ObjectMeta.Annotations) != 2 {
 		t.Errorf("Wrong number of annotations")
 	}
+	if len(job.ObjectMeta.Annotations) != 0 &&
+		job.ObjectMeta.Annotations[batchv1.JobCreationTimestampAnnotation] != scheduledTime.String() {
+		t.Errorf("Wrong job creation timestamp annotation")
```

Comment:
Unit test seems to be failing as the scheduledTime isn't what is set on the above appened func but the creationtimestamp of the batch job.  

The scheduledTime in seconds gets appended to the name of the job but the creationTimestamp seem to be different than what is passed in.

Reply:
So the Annotation isn't going to equal the string of the scheduledTime being that this will be different than what is set when creating the job.

Reply:
This change should work.

Reply:
The annotation will have the original scheduled time, the `scheduledTime` will be set as the new value for the`CreationTimestamp` meta and the user doesn't have a way to get the original/expected scheduled time.